platform/email/platform-email-impl/src/test/java/org/codice/ddf/platform/email/impl/SmtpClientImplITCaseTest.java

```diff
@@ -111,9 +114,10 @@ public void testSendWithAttachments()
 
     server.stop();
 
-    assertThat(server.getReceivedEmailSize(), is(1));
```

Comment:
Open to doing some of this better. I was just looking to do the minimum needed to upgrade the test depenendency